.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  install-dependencies:
+    docker:
+      - image: circleci/node:8
+    working_directory: ~/insight-website
```

Comment:
Why don't we use the default working directory?

Reply:
Now I use the default directory (~/project). I did not have good reason to do that.